src/const.js

```diff
@@ -73,7 +73,10 @@ export const LOCAL_PROTOCOLS = ['chrome:', 'resource:'];
 
 export const MANIFEST_JSON = 'manifest.json';
 
-export const VALID_MANIFEST_VERSION = 2;
+export const MANIFEST_VERSION_VALID = 2;
```

Comment:
It's the top of the patch so I will probably find out later but this constant does not speak to me. The other constants make sense but I don't know what that one should do.

If it is about validation and we want to introduce new manifest versions, I would have expected an array instead. OTOH we have min/max already...

Reply:
@willdurand MANIFEST_VERSION_VALID was meant to go away (I originally renamed it from VALID_MANIFEST_VERSION to MANIFEST_VERSION_VALID for consistency with some of the other consts, but in a later refactoring I actually opted for MANIFEST_VERSION_DEFAULT + MANIFEST_VERSION_MIN / MANIFEST_VERSION_MAX:

- MANIFEST_VERSION_DEFAULT is the default we would use if we didn't got one explicitly and we need exactly one manifest version to refer to, e.g.
  - in src/schema/validator.js: if the current extension being validate does miss a manifest_version property, we will complete the validation considering the manifest_version as if it was set to MANIFEST_VERSION_DEFAULT (but this doesn't mean we wouldn't want for a missing manifest_version mandatory property, that would still happen as expected)
  - in src/schema/browser-api.js: same as for validator.js, we would validate the API calls as it the manifest_version missing was meant to be set to the default version (but we will still collect an error for the missing manifest_version property) 
- MANIFEST_VERSION_MIN and MANIFEST_VERSION_MAX are the default values for the `min/max_manifest_version` json schema definitions that may not have one on their own.

Locally I've just added a fixup patch that removes MANIFEST_VERSION_VALID and does add inline comments on these consts to point out what they mean / used for, that change will be pushed along the next update on this PR.